lib/datasource/clojure/index.ts

```diff
@@ -1,8 +1,26 @@
+import { Datasource } from '../datasource';
+import { getReleases } from '../maven';
 import { MAVEN_REPO } from '../maven/common';
+import { GetReleasesConfig, ReleaseResult } from '../types';
 
 export const id = 'clojure';
 export const customRegistrySupport = true;
 export const defaultRegistryUrls = ['https://clojars.org/repo', MAVEN_REPO];
 export const registryStrategy = 'merge';
 
-export { getReleases } from '../maven';
+export class Clojure extends Datasource {
+  readonly id = 'clojure';
+
+  readonly registryStrategy = 'merge';
+
+  readonly customRegistrySupport = true;
+
+  readonly defaultRegistryUrls = ['https://clojars.org/repo', MAVEN_REPO];
```

Comment:
do we need those duplicates?

Reply:
This is an issue I don't yet have a great solution for. If I look at `id`, it's defined in the `DatasourceApi` interface, and I defined it as `abstract id` in the abstract class `Datasource` so that subclasses are required to implement it. However that means that you need to instantiate `ClojureDatasource` in order to read its id.

Maybe dropping the `abstract` modifier, and just making it `static`? [Here's a TS playground link to highlight the issue](https://www.typescriptlang.org/play?#code/IYIwzgLgTsDGEAJYBthjAgIsCaD2ArlLAKYIDeAUAjQpDgJawJQnAAmeAdsgJ4IN2ALjrQGXAOaUAvpUoo0GAMLI8AKyIlsuMIWJkSADwgku7DNvxFSFarXoQmA9ggC8CAOQp1mjwG4ZOQV0BAAxPCgJPAhLXWsDY1NzLBwrfVtA+W5dZBIAOlUJAAoVH1ZYvVI8wQBKLK4c-MKi8Mjoivjq9hqgA)

Reply:
Or just instantiating the class?